oauth2client/flask_util.py

```diff
@@ -331,6 +333,20 @@ def _make_flow(self, return_url=None, **kwargs):
             redirect_uri=url_for('oauth2.callback', _external=True),
             **kw)
 
+        flow_key = 'google_oauth2_flow_{0}'.format(csrf_token)
+        session[flow_key] = pickle.dumps(flow)
```

Comment:
pickle's only used for the flow because it doesn't implement ``to_json`` and ``from_json``. I think it should, but it might be out of the scope of this bug fix. Maybe something to follow up on?